### ISSUE TITLE
Errors

### ISSUE THREAD

**Issue by osbornegames** [MilkAndBanana01/godot-fps-template#2]
I keep getting "Index p_surface = 0 is out of bounds" errors every time I pull up the project in Gadot. These errors seem to make it where I can't see nearly any textures or objects and I can't even playtest the game. By the way love this project and I am glad you are helping out people like me that are new to the engine thanks!

**Comment by MilkAndBanana01** [comment]
I have tried to clone the repo and run it  in my laptop. It works fine on my end. You can download the latest one since I do give out updates here and there.

**Comment by osbornegames** [comment]
If I can get this issue out there I figure maybe someone can look at it. I wonder if the problem may be that the file is in the wrong directory? Not sure. Here are some screenshots
![Screenshot (39)](https://user-images.githubusercontent.com/97578237/149640867-a1120b72-011f-414d-9b35-b4fb46e12221.png)
![Screenshot (40)](https://user-images.githubusercontent.com/97578237/149640869-64f4dd5c-425e-4c25-b63c-4042dee2aee1.png)

**Comment by Eris7090** [comment]
> If I can get this issue out there I figure maybe someone can look at it. I wonder if the problem may be that the file is in the wrong directory? Not sure. Here are some screenshots ![Screenshot (39)](https://user-images.githubusercontent.com/97578237/149640867-a1120b72-011f-414d-9b35-b4fb46e12221.png) ![Screenshot (40)](https://user-images.githubusercontent.com/97578237/149640869-64f4dd5c-425e-4c25-b63c-4042dee2aee1.png)

This is what it looks like on my machine as well

**Comment by osbornegames** [comment]
> > If I can get this issue out there I figure maybe someone can look at it. I wonder if the problem may be that the file is in the wrong directory? Not sure. Here are some screenshots ![Screenshot (39)](https://user-images.githubusercontent.com/97578237/149640867-a1120b72-011f-414d-9b35-b4fb46e12221.png) ![Screenshot (40)](https://user-images.githubusercontent.com/97578237/149640869-64f4dd5c-425e-4c25-b63c-4042dee2aee1.png)
> 
> This is what it looks like on my machine as well

Yeah I still have this issue. Hopefully I can get someone to help me look at it, as this template is the best one I've seen yet imo
